static/js/components/plot/VegaPhotometry.jsx

```diff
@@ -231,8 +236,15 @@ const VegaPhotometry = (props) => {
   }
 
   let photometryFiltered = photometry;
+
+  photometryFiltered = photometryFiltered.filter(
```

Comment:
maybe the git diff is not helping, but in the VegaPhotometry component, are you **always** hiding the swiftxrt data, regardless of the magsys?

Reply:
oh @camilledzt vega in this context is just the name of the plotting library, has nothing to do with the magnitude system! This component here is completly agnostic to the magsys values